pkg/broker/registry.go

```diff
@@ -125,6 +127,17 @@ func (svc *BrokerService) UserDefinedPlansProperty() string {
 	return fmt.Sprintf("service.%s.plans", svc.Name)
 }
 
+// TileUserDefinedPlansVariable returns the name of the user defined plans
+// variable for the broker tile.
+func (svc *BrokerService) TileUserDefinedPlansVariable() string {
+	v := utils.PropertyToEnvUnprefixed(svc.Name)
+	if strings.HasPrefix(v, "GOOGLE_") {
+		v = v[7:]
```

Comment:
This seems brittle. Consider instead:
prefix := "GOOGLE_"
if strings.HasPrefix(v, prefix) {
  v = v[len(prefix):]
}